### ISSUE TITLE
Implement UI Design for All Core Pages

### ISSUE THREAD

**Issue by ariunaamy** [ariunaamy/react-web-raffle#1]
Construct simple and functional UIs for participant registration and winner selection within a single day, adhering to provided wireframes with minimal design enhancements.

1. **Participant Registration UI:** Build a straightforward form for participant registry.
2. **Winner Selection UI**: Set up a basic interface for selecting winners from registered participants.
3. **Focus on Essentials**: Ensure core functionality, responsiveness, and basic accessibility.

Requirements
Utilize [Flowbite styled components](https://flowbite-react.com/) to expedite development.
Base UIs on provided [Wireframes](https://github.com/pursuit-nw-accelerator/fullstack-takehome-challenge/blob/main/Web-Raffle-App-Wireframes.pdf).
Prioritize load efficiency and straightforward interaction.

Total Development: 1 day

Ensure all wireframes, assets, and guidelines are ready before starting.

Confirm backend readiness for seamless integration.

**Comment by mikeboyle** [comment]
The scope of this ticket is too big for my preferences. There are two pages listed here, so I would make a separate ticket for each page. Even then, some pages might need to be broken into more tickets if they are complicated or have more parts.

At work, it's good to write smaller tickets so that:
- we can demonstrate we're making progress more frequently
- different team members can work on tickets in parallel. For example, one person could be building the participant registration form at the same time as another person is building the choose winner form.
